benches/reconstruct.rs

```diff
@@ -1,17 +1,22 @@
-#![feature(test)]
+#![cfg_attr(nightly, feature(test))]
```

Comment:
I'm wondering if we should just remove it entirely as what you have implemented with criterion seems to do the same and more, is that correct?

Reply:
That is correct. This PR covers reconstruction. I suggest removing `benches/reconstruct.rs`.

Reply:
Let's do it then